core/templates/dev/head/pages/exploration_player/LearnerLocalNav.js

```diff
@@ -38,15 +38,22 @@ oppia.controller('LearnerLocalNav', [
         resolve: {},
         controller: [
           '$scope', '$uibModalInstance', '$timeout', 'PlayerPositionService',
-          'ExplorationPlayerService',
+          'ExplorationPlayerService', 'IS_CURRENT_RTE_CKEDITOR',
           function(
               $scope, $uibModalInstance, $timeout, PlayerPositionService,
-              ExplorationPlayerService) {
+              ExplorationPlayerService, IS_CURRENT_RTE_CKEDITOR) {
             var stateName = PlayerPositionService.getCurrentStateName();
             $scope.originalHtml = ExplorationPlayerService.getStateContentHtml(
               stateName);
+            $scope.isCurrentRTECKEditor = IS_CURRENT_RTE_CKEDITOR;
             $scope.description = '';
-            $scope.suggestionHtml = $scope.originalHtml;
+            // If there is a ng-model, you have to have a dot in there.
+            // Reference https://stackoverflow.com/questions/12618342/ng
+            // -model-does-not-update-controller-value/22768720#22768720.
+            // Due to this reason, suggestionHtml has been changed to
+            // suggest.suggestionHtml.
+            $scope.suggest = {};
```

Comment:
Have a better name for this -- suggestionData perhaps? ("suggest" is an action, so not appropriate as a noun here). 

Also set the object's keys directly on-declaration, otherwise it becomes harder to tell what its contents are at any given time. Don't add new keys beyond the ones initially set. (This is why we typically use domain objects -- to avoid this sort of thing.)